src/installer/pkg/archives/dotnet-nethost.proj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
+    <CreateSymbolsArchive>true</CreateSymbolsArchive>
```

Comment:
Source-build doesn't strip the host binaries, so don't create a symbols archive when source-building.
```suggestion
    <CreateSymbolsArchive Condition="'$(DotNetBuildFromSource)' != 'true'">true</CreateSymbolsArchive>
```